mlflow/models/evaluation/base.py

```diff
@@ -22,14 +22,6 @@
 _logger = logging.getLogger(__name__)
 
 
-class EvaluationMetrics(dict):
-    """
-    A dictionary of model evaluation metrics.
-    """
-
-    pass
-
-
 class EvaluationArtifact(metaclass=ABCMeta):
```

Comment:
@WeichenXu123 Can we make `EvaluationArtifact.load()` and `EvaluationArtifact.save()` private by adding an underscore? Right now, they show up in the API docs, which may be confusing for most users.